flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/container/RedisClusterContainer.java

```diff
@@ -47,14 +47,7 @@ public RedisClusterContainer(JedisCluster jedisCluster) {
     }
 
     @Override
-    public void open() throws Exception {
-
-        // echo() tries to open a connection and echos back the
-        // message passed as argument. Here we use it to monitor
-        // if we can communicate with the cluster.
-
-        jedisCluster.echo("Test");
-    }
+    public void open() throws Exception {}
```

Comment:
why the "jedisCluster.echo("Test")" is removed?

this could create any colateral? 
this is more an optimization than a need from the bump version no?